lang/libgcc/Portfile

```diff
@@ -3,9 +3,9 @@
 PortSystem          1.0
 PortGroup select    1.0
 
-epoch               3
+epoch               4
```

Comment:
not sure why we are changing the epoch here. that is usually done when going down in a version, and we’re going up.

Reply:
OK, I'll revert this. The `version` is bumped anyway.